benchmarks/benchmarks.c

```diff
@@ -55,3 +54,188 @@ void benchmarkFillup ()
 		}
 	}
 }
+
+#ifdef ELEKTRA_ENABLE_OPTIMIZATIONS
+
+// needed for the opmphmRandom ()
+#include "../src/libs/elektra/opmphm.c"
+
+/**
+ * @brief Fills a string up with random chars and null terminates it.
+ *
+ * @param name the string
+ * @param length the length of the string (excluding the '\0')
+ * @param seed to generate random data
+ *
+ */
+static void fillUpWithRandomChars (char * name, int length, uint32_t * seed)
+{
+	int i = 0;
+	char * c = name;
+	while (i < length)
+	{
+
+		if ((opmphmRandom (seed) % 10) == 0)
```

Comment:
More bits are mostly needed for the sequence length. The lrand48 (which we can not use, see above) also only returns values from 0 and 2^31.